setup.cfg

```diff
@@ -42,9 +42,10 @@ line_length = 88
 ignore=D100,D104,D105,D200,D203,D400
 
 [prequ]
+annotate = yes
 requirements =
     django-environ
-	Django~=2.2.9
+	Django~=2.2.12
```

Comment:
I feel like this django-helusers commit should be split up into multiple commits. Now there is django updates, package additions, migrations and settings all in the same commit. Would be much nicer to have atleast the django update and prequ stuff on its own.

Reply:
Good point, I'll change this.